### PR TITLE
Update Czech localization

### PR DESCRIPTION
**Pull request by ferben**

### CODE REVIEW

File: Translations/cs.lproj/InternetAccessPolicy.strings [hunk 1]
```diff
@@ -13,5 +13,5 @@
 "ApplicationDescription" = "Keka je archivační aplikace pro macOS.";
 
 /* Software update connection: */
-"SoftwareUpdatePurpose" = "Keka vyhledává aktualizace připojením k u(pdate).keka.io.";
+"SoftwareUpdatePurpose" = "Keka vyhledává aktualizace pomocí připojení k serveru .keka.io.";
```
Comment (aonez) [inline Translations/cs.lproj/InternetAccessPolicy.strings:16]: The actual server is _u.keka.io_ or _update.keka.io_, that's why it must say _u(pdate).keka.io_

**Review by aonez — CHANGES_REQUESTED** [review]
Thank you so much @ferben!

Please check the comment about the _u(pdate).keka.io_ string, a minor fix.

Also if you want to appear in the translators credits don't forget to add yourself there:
https://github.com/aonez/Keka/blob/912838cc96266446944fa702241a4660b4b33b4a/Translations/Common/Translators.html#L47-L51

File: Translations/cs.lproj/InternetAccessPolicy.strings (hunk 1 above)
Comment (ferben) [reply Translations/cs.lproj/InternetAccessPolicy.strings:16]: OK, I'll fix it.

Comment (aonez) [reply Translations/cs.lproj/InternetAccessPolicy.strings]: Thanks again @ferben!

**Comment by aonez** [comment]
Since it has a lot of change, here an updated build in case you want to check it: [Keka-v1.2.62.r5168](https://github.com/aonez/Keka/releases/download/dev-test-builds/Keka-v1.2.62.r5168.7z)

**Comment by ferben** [comment]
Thank you, I'll make a review 👍

**Pull request merged**